### DISCUSSION TITLE
Newsletter support

### DISCUSSION THREAD

**Discussion by jtebert** [wagtail/wagtail#8326] [Ideas]
I'm not sure if this falls within the goals of Wagtail, but I think it would be fantastic for it to support newsletters: creating them like you would create a page (including setting a date for them to go live - e.g. be sent) and managing subscriptions similar to forms or users.

**Comment by davecranwell** [comment]
Given the superb offering by companies such as Campaign Monitor, Mail Chimp et al, I wouldn't say that Wagtail should, or would be able to compete in that arena. CRM integration is however something we've talked about before and Campaign Monitor etc are essentially CRMs with a very specific focus (mailing). I expect we'd probably tackle this from the perspective of making integration with third parties like Campaign Monitor easier, rather than trying to directly compete in an already busy market.

**Comment by chimeno** [comment]
I would same the same as @davecranwell , nowadays CMS's should have good integration with existing services in order to be able to compete in price and time with others.

The question will be:
- ¿Should wagtail make  a contrib app for newsletters (integrated with other services), or are the developers implement the specific situations?

**Comment by gasman** [comment]
I think this would be a very good fit for either a contrib app, or a separately-installed Wagtail add-on module.

**Comment by jtebert** [comment]
Integration with existing services would be great. For my use case, I'm looking to automatically generate weekly emails of articles on my site, so being able to connect my models to a company's newsletter management would work well.

**Comment by simo97** [comment]
Hi, am just discovering this thread, for me we could have a wagtail app which can be integrated with other third party services (Mailchimps, MailJet, etc), and allow the CMS user to have some basics actions available from the CMS's dashboard such as , view the mailing list, trigger email sending for example.

**Comment by marteinn** [comment]
https://github.com/neon-jungle/wagtail-birdsong should cover some of the use cases mentioned in this thread.

**Comment by thibaudcolas** [comment]
Converting this to a Discussion because I believe we’re very unlikely to support newsletters directly in `wagtail/wagtail`.
